### PR TITLE
Update theming docs after #9698

### LINKED ISSUE #10146: [docs] Theming dev docs outdated
## Describe the bug
Theming dev docs are outdated since https://github.com/owncloud/web/pull/9698 has been merged.

## Steps to reproduce
1. Visit https://owncloud.dev/clients/web/theming/
2. Try to follow the docs for building themes

## Expected behavior
Instructions for building themes are correct.

## Actual behavior
Nothing works anymore. Be confused.

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Related Issue
- Fixes #10146

### CODE REVIEW

File: docs/theming/_index.md [hunk 1]
```diff
@@ -25,60 +25,136 @@ To reference your theme, you have two options:
...
+    "web": {}
+  }
+}
+```
+
+### Common section
+
+The `common` section provides a set of information that is designed to be available for all clients. It gets merged "down" to the final themes and aims to reduce duplication, but can be overwritten by more specific information inside both the clients' defaults and actual themes.
+
+The structure of a valid `comon` section is outlined below:
```
Comment (kulmann) [inline docs/theming/_index.md:52]:
```suggestion
The structure of a valid `common` section is outlined below:
```

File: docs/theming/_index.md [hunk 2]
```diff
@@ -25,60 +25,136 @@ To reference your theme, you have two options:
...
+  "logo": "themes/owncloud/assets/logo.svg",
+  "urls": {
+    "accessDeniedHelp": "",
+    "imprint": "",
+    "privacy": ""
+  }
+}
+```
+
+- `name` specifies the publicly visible name for this oCIS instance
```
Comment (kulmann) [inline docs/theming/_index.md:67]: My feeling is that the repetition of `for this oCIS instance` is a bit too much.

File: docs/theming/_index.md (hunk 1 above)
Comment (pascalwengerter) [reply docs/theming/_index.md]: C'mon :trollface:

Comment (kulmann) [reply docs/theming/_index.md]: Was thinking the same :trollface:

**Review by kulmann — APPROVED** [review]
nice, thank you 👍

**Pull request merged**